src/Env.jsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { Button } from "@patternfly/react-core/dist/esm/components/Button";
+import { FormGroup } from "@patternfly/react-core/dist/esm/components/Form";
+import { FormHelper } from "cockpit-components-form-helper.jsx";
+import { Grid } from "@patternfly/react-core/dist/esm/layouts/Grid";
+import { TextInput } from "@patternfly/react-core/dist/esm/components/TextInput";
+import { TrashIcon } from '@patternfly/react-icons';
+import cockpit from 'cockpit';
+
+import * as utils from './util.js';
+
+const _ = cockpit.gettext;
+
+export function validateEnvVar(env, key, validationFailed) {
+    const delta = { ...validationFailed };
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1354-20230912-120439-9f74a06a-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)